### PR TITLE
App Navigation

### PR DESCRIPTION
**Pull request by allisonbhavsar**
### Administrative Info
Monday Board ID: 853642866
Make sure your branch name conforms to: `<feature/staging/hotfix/...>/[username]/[Monday Item ID]-[3-4 word description separated by dashes]`. Otherwise, please rename your branch and create a new PR.

### Changes
What changes did you make?
- Set up routing using React Navigation
- Created bottom menu linking to Home, Chat, and Settings Screens

### Testing
How did you confirm your changes worked? 
- Navbar rendered and routed properly on two different devices

### Confirmation of Change 
Upload a screenshot, if possible. Otherwise, please provide instructions on how to see the change.

IMG_CF7984FB0146-1

### CODE REVIEW

File: mobile/App.tsx [hunk 1]
```diff
@@ -1,20 +1,79 @@
...
-    </View>
+    <NavigationContainer>
+      <Tab.Navigator
+        initialRouteName="Home"
+        screenOptions={({ route }) => ({
+          tabBarIcon: ({ focused }) => {
+            let icon;
+            const tint = focused ? '#E89228' : '#B8B8B8';
+            let width;
+            let height;
```
Comment (dacruz21) [inline mobile/App.tsx:21]: Let's define these styles in the style sheet below, just to keep things a little cleaner.

File: mobile/App.tsx [hunk 2]
```diff
@@ -1,20 +1,79 @@
...
+              icon = require('./assets/images/Cog.png');
+              width = 26;
+              height = 26;
+            } else if (route.name === 'Chat') {
+              icon = require('./assets/images/Chat_bubble.png');
+              width = 22;
+              height = 22;
+            }
+
+            return <Image style={{ width: width, height: height, tintColor: tint }} source={icon} />;
```
Comment (dacruz21) [inline mobile/App.tsx:37]: Also this is kinda messy. How about a map that you can index into with `route.name`? like:

```ts
const TabIcons = {
  Home: <Image style={styles.homeTabIcon} source={require('./assets/images/Home.png')} />,
  ...
};
```

```ts
screenOptions = {({ route }) => ({
  tabBarIcon: () => TabIcons[route.name]
)}
```

File: mobile/package.json [hunk 3]
```diff
@@ -1,11 +1,19 @@
...
     "expo": "^40.0.0",
     "react": "16.13.1",
     "react-dom": "16.13.1",
     "react-native": "^0.63.4",
+    "react-native-gesture-handler": "~1.8.0",
+    "react-native-reanimated": "~1.13.0",
+    "react-native-safe-area-context": "3.1.9",
     "react-native-screens": "~2.15.0",
+    "react-native-svg": "^12.1.0",
+    "react-native-svg-transformer": "^0.14.3",
```
Comment (dacruz21) [inline mobile/package.json:16]: Just checking... are all of these needed? At the very least I don't think we need the svg libs now that we're using PNGs, right? (also if you remove any remember to `npm install` again so the package lock gets updated)

File: mobile/src/screens/ChatScreen.tsx [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { Text, View } from 'react-native';
+
+export const ChatScreen: React.FC = () => {
+  return (
+    <View>
+      <Text>Chat</Text>
+    </View>
+  );
+};
```
Comment (dacruz21) [inline mobile/src/screens/ChatScreen.tsx:10]: Super minor thing but can you just add a quick stylesheet so the view is centered in the screen (like the original TODO in App.tsx)? Same with Home and Settings

File: mobile/src/screens/Settings.tsx [hunk 5]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { Text, View } from 'react-native';
+
+export const Settings: React.FC = () => {
```
Comment (dacruz21) [inline mobile/src/screens/Settings.tsx:4]: I like the convention of ending each screen with `Screen`, can you add that here?

**Review by typokign — CHANGES_REQUESTED** [review]
Nice! Let's just clean up the logic for the tab navigator and a few other minor things. Also, can you put comments on the 3 screens? Just something very brief, like "Left tab on home screen for chatting with volunteers".

**Review by typokign — APPROVED** [review]
lgtm

**Review by RohithKasar — APPROVED** [review]
Code base looks clean. Ran it on my phone and was able to switch through the 3 navigation items at the bottom. Colors and highlighting are nice!

looks good to me

**Pull request merged**